libraries/AP_Terrain/AP_Terrain.cpp

```diff
@@ -309,12 +304,9 @@ void AP_Terrain::update(void)
     // just schedule any needed disk IO
     schedule_disk_io();
 
-    // try to ensure the home location is populated
-    float height;
-    height_amsl(ahrs.get_home(), height, false);
```

Comment:
I'll leave it in, since it's controversial, but again. current_loc is darned close to home and will ensure that the location is loaded before takeoff, unless you move home. But again will just leave it alone for now.